splash/qtrender_lua.py

```diff
@@ -771,24 +771,41 @@ def _assert_number(value, name):
 
     @command()
     def mouse_click(self, x, y):
-        self._check_mouse_coordinantes(x, y)
+        self._check_mouse_coordinates(x, y)
         self.tab.mouse_click(x, y)
 
     @command()
     def mouse_press(self, x, y):
-        self._check_mouse_coordinantes(x, y)
+        self._check_mouse_coordinates(x, y)
         self.tab.mouse_press(x, y)
 
     @command()
     def mouse_release(self, x, y):
-        self._check_mouse_coordinantes(x, y)
+        self._check_mouse_coordinates(x, y)
         self.tab.mouse_release(x, y)
 
     @command()
     def mouse_hover(self, x, y):
-        self._check_mouse_coordinantes(x, y)
+        self._check_mouse_coordinates(x, y)
         self.tab.mouse_hover(x, y)
 
+    @command()
+    def click(self, selector):
+        element = self.tab.find_element(selector)
+        if element.isNull():
+            raise ScriptError({
+                "argument": selector,
+                "message": "element at {!r} not found".format(selector)
+            })
+        x, y = element.geometry().x(), element.geometry().y()
+        return self.mouse_click(x, y)
```

Comment:
I can't seem to make this exception reach up if click is called from another command. You can see one test failing checking for that. Ideas?